src/openforms/forms/api/viewsets.py

```diff
@@ -817,6 +817,7 @@ class CategoryViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = Category.objects.all()
     serializer_class = CategorySerializer
     # TODO this permission isn't ideal with its mix of public and write access
+    authentication_classes = (TokenAuthentication, SessionAuthentication)
     permission_classes = [FormAPIPermissions]
```

Comment:
I think a separate permission class for this that checks if you have explicit `view_category` or `FormAPIPermissions` seems appropriate

I think in DRF these days you can do `Permission1 | Permission2` to do an OR

Reply:
But wouldn't this be problematic for the POST actions?

Reply:
it's a `ReadOnlyModelViewSet` :)

but it's a good idea to add some tests that you get an HTTP 405 on write actions through the API, yes.

Reply:
oh oups, silly me :see_no_evil: